core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1445,6 +1478,21 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
   }
 
+  def updateSubmissionUserComment(workspaceName: WorkspaceName, submissionId: String, newComment: UserCommentUpdateOperation): Future[PerRequestMessage] = {
+    validateMaxStringLength(newComment.userComment, "userComment", 1000)
+
+    getWorkspaceContextAndPermissions(workspaceName, SamWorkspaceActions.write) flatMap { workspaceContext =>
+      dataSource.inTransaction { dataAccess =>
+        withSubmissionId(workspaceContext, submissionId, dataAccess) { submissionId =>
+          dataAccess.submissionQuery.updateSubmissionUserComment(submissionId, newComment.userComment) map { rows =>
+            if (rows == 1) RequestComplete(StatusCodes.NoContent)
```

Comment:
What does "rows" correspond to?

Reply:
"rows" indicates how many records were updated by the update query in `updateSubmissionUserComment`. It should only be 1 since each submission can only have 1 associated userComment

Reply:
could be called `rowsUpdated` to answer @cahrens's question via the code itself 😃